ksql-examples/src/main/java/io/confluent/ksql/datagen/AvroConsumer.java

```diff
@@ -58,10 +60,15 @@
       Map<String, Object> serdeProps = new HashMap<>();
       serdeProps.put(KsqlGenericRowAvroSerializer.AVRO_SERDE_SCHEMA_CONFIG, schemaStr);
 
-      final Serializer<GenericRow> genericRowSerializer = new KsqlGenericRowAvroSerializer(null);
+      final Serializer<GenericRow> genericRowSerializer = new KsqlGenericRowAvroSerializer(null,
+                                                                                           null,
+                                                                                           null,
+                                                                                           false);
       genericRowSerializer.configure(serdeProps, false);
 
-      final Deserializer<GenericRow> genericRowDeserializer = new KsqlGenericRowAvroDeserializer(null);
+      final Deserializer<GenericRow> genericRowDeserializer = new KsqlGenericRowAvroDeserializer(null,
```

Comment:
Same here, i believe this will result iin an NPE

Reply:
Yes, both avro producer and consumer are not used. I will update the examples package code based on the new implementation in another PR.

Reply:
So these classes should be deleted? Apart from the obvious NPEs, it is also curious that you are passing a `MockSchemaRegistryClient' in a non test class. 

I would not be comfortable committing these changes to these classes since they are really odd. Would it be possible to remove these classes in this PR itself?